ENIGMAsystem/SHELL/Bridges/Win32-Direct3D11/graphics_bridge.cpp

```diff
@@ -146,7 +145,6 @@ void WindowResized() {
 }
 
 void EnableDrawing(void* handle) {
-  get_window_handle();
```

Comment:
You should always be skeptical of deleting a line that you think doesn't do anything. Usually, that's a code smell. In this case, the comment you modified below is clear about this: "This function is called by d3d and dsound to ensure the window handle is set"

If this were done properly, the author would have made a new routine (in Mandatory or in the Win32 bridge header) to make sure that the window was initialized and ready to go. Instead, we ended up with this. But deleting this line will break this system.

Reply:
This line in fact doesn't do anything after the changes I made. get_window_handle() has been renamed to window_id_init(); the two functions do the same exact thing. Only now instead of it being called from direct sound and d3d it is being called before both of those things initialized, under initGameWindow() instead, so instead of calling it in multiple irrelevent places, we are calling it once in a relevant place.